plugins/postgresql_alt.py

```diff
@@ -1,24 +1,37 @@
 # -*- coding: utf8 -*-
 
+import os
+
+from shlex import split as shlex_split
 from sos.report.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
-from typing import List, Tuple
+from subprocess import check_output, CalledProcessError
+from typing import List, Optional, Tuple
 
 import psycopg2
 
 DEFAULT_DSN = 'postgresql://postgres@localhost/postgres'
 
+
+class LoggingInfo:
+    def __init__(self, collect_logs, log_dir, data_dir):
+        self.collect_logs = collect_logs
+        self.log_dir = log_dir
+        self.data_dir = data_dir
+
+
 class PostgreSQLAlt(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
     """PostgreSQL alternative collection plugin"""
     plugin_name = "postgresql_alt"
     requires_root = False
     short_desc = 'PostgreSQL alternative collection plugin'
 
     option_list = [
-        ('dsn', 'The PostgreSQL DSN to collect information from.', '', DEFAULT_DSN)
+        ('dsn', 'The PostgreSQL DSN to collect information from.', '', DEFAULT_DSN),
+        ('container_id', 'The docker container id where PostgreSQL runs in.', '', '')
```

Comment:
Via sosreport, it should list the container ID in the options.

Reply:
Understood, thank you. 
Please document how to use the -k switch to provide dsn and container_id. 
It would also be helpful to mention that the logging_collector should be turned.